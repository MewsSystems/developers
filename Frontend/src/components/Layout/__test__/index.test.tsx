import { Layout } from '../'
import React from 'react'
import { shallow } from 'enzyme'

describe('Test Layout', () => {
  it('Tests snapshot', () => {
    expect(shallow(<Layout>test</Layout>)).toMatchSnapshot()
  })
})
