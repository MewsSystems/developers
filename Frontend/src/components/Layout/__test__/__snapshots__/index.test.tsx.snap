// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Layout Tests snapshot 1`] = `
<div>
  <styled.div>
    <Header />
    <Spacing
      outer="0 0 54px;"
    >
      test
    </Spacing>
  </styled.div>
  <Footer />
</div>
`;
